samples/sample_plugins/vpp_plugin/include/mfx_multi_vpp.h

```diff
@@ -33,13 +33,13 @@ class MFXVideoMultiVPP : public MFXVideoVPP
 
     // topology methods
     virtual mfxStatus QueryIOSurf(mfxVideoParam *par, mfxFrameAllocRequest request[2], mfxVideoParam *par1 = NULL, mfxVideoParam *par2 = NULL)
-    { par1; par2; return MFXVideoVPP_QueryIOSurf(m_session, par, request); }
+    { (void)par1; (void)par2; return MFXVideoVPP_QueryIOSurf(m_session, par, request); }
```

Comment:
Do we remove (void)par1 or use /*par1*/? Please, decide. Right now changes are not consistent.